src/utils/MessageBuilder.ts

```diff
@@ -138,11 +152,16 @@ export default class MessageBuilder {
       color: "GREEN",
       thumbnail: { url: this.normIconURL },
     });
-    const pickTeamsButton = new MessageButton({
+    const randomTeamsButton = new MessageButton({
       customId: ButtonCustomID.CreateRandomTeam,
       label: "Random",
       style: "PRIMARY",
     });
+    const pickTeamsButton = new MessageButton({
```

Comment:
Can we rename this variable to `pickCaptainsButton` or something more appropriate?